src/Psy/Plugin/Manager.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace Psy\Plugin;
+
+class Manager
+{
+    /** @var AbstractPlugin[] */
+    protected static $plugins = array();
+
+    /**
+     * @param AbstractPlugin $plugin
+     * @param $name
+     */
+    public static function register(AbstractPlugin $plugin, $name)
+    {
+        self::$plugins[$name] = $plugin;
+    }
```

Comment:
I'm not quite sure that this class should be static (Or otherwise it should be called something else, such as `PluginRegistry` ?) ; thus, we would need to pass the `Manager` instance to the `register` method of the Plugin